src/SDL2.cs

```diff
@@ -1648,23 +1814,39 @@ public static string SDL_GetWindowTitle(IntPtr window)
 
 		/* IntPtr refers to a function pointer */
 		[DllImport(nativeLibName, EntryPoint = "SDL_GL_GetProcAddress", CallingConvention = CallingConvention.Cdecl)]
-		private static extern IntPtr INTERNAL_SDL_GL_GetProcAddress(
-			byte[] proc
+		private static extern unsafe IntPtr INTERNAL_SDL_GL_GetProcAddress(
+			byte* proc
 		);
-		public static IntPtr SDL_GL_GetProcAddress(string proc)
+		public static unsafe IntPtr SDL_GL_GetProcAddress(string proc)
```

Comment:
There's another overload on line 1854 for SDL_GL_GetProcAddress(IntPtr proc) used by the Modern and OpenGL graphics devices. This causes trouble for the BRUTE path.

Reply:
I think this is handled in our branch - usually it’s just a duplicate extern with the wrong pointer overload. Worst case this can just be removed, since IGLDevice is being removed as well.

Reply:
Refering to the BRUTE/fna-next branch?  I'm on that branch.

Reply:
For now cover everything except this function - I’ll investigate this myself once everything is ready to test.

Reply:
Other than the opportunity to optimize where strings shouldn't be null, I think this ones ready.  I'm going to do some builds/smoke-tests.